pkg/cmd/pulumi-language-java/main.go

```diff
@@ -447,6 +504,16 @@ func (host *javaLanguageHost) RuntimeOptionsPrompts(_ context.Context,
 	return &pulumirpc.RuntimeOptionsResponse{}, nil
 }
 
+func WaitForDebuggerReady(out io.Reader) error {
```

Comment:
A doc comment on this might be good. Is the string the same regardless of whether we use Maven, etc.?